com.unity.ml-agents/Documentation~/com.unity.ml-agents.md

```diff
@@ -124,3 +129,4 @@ Please refer to "Information that is passively collected by Unity" in the
 [execution order of event functions]: https://docs.unity3d.com/Manual/ExecutionOrder.html
 [connect with us]: https://github.com/Unity-Technologies/ml-agents#community-and-feedback
 [ml-agents forum]: https://forum.unity.com/forums/ml-agents.453/
+[ML-Agents GitHub repo]: https://github.com/Unity-Technologies/ml-agents/blob/release_12_docs/com.unity.ml-agents.extensions
```

Comment:
@chriselion - I hardcoded the release version path (release_12_docs) similar to other links. How/when do we update with each release?

Reply:
There's a script on release_13_branch that automates the updating of those links